assets/sparksql/main.py

```
@@ -0,0 +1,84 @@
+        Ingests the dataset at self.datasets_path
+        """
+        self.docker_execute([
+            f"python3 {ASSETS_DIR}/ingest.py {self.datasets_path} {SPARKSQL_STORAGE}"
+            ])
+    
+    def search(self, query):
+        """
+        Searches an already-ingested dataset with query, which is populated within config.yaml
+        """
+        return self.docker_execute([
+            f"python3 {ASSETS_DIR}/search.py {query} {SPARKSQL_STORAGE}"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)